### ISSUE TITLE
Regarding training yolov3 for custom dataset

### ISSUE THREAD

**Issue by sharmarochan** [AlexeyAB/darknet#3934]
Hi Alexey,
I am training yolov3 model from scratch for object detection and tracking for my custom dataset, the images are approx 2100 which contains 16 classes in total.
I have already trained for 20622 iterations(on gpu) but my loss is not going down beyond 1.5, I have attached my configuration file, just have a look if I'm going wrong anywhere.
[cfg.txt](https://github.com/AlexeyAB/darknet/files/3620349/cfg.txt)

**Comment by AlexeyAB** [comment]
What Loss and mAP do you get?

**Comment by sharmarochan** [comment]
Average Loss in the chart.png shows (1.5). I have tried using adam optimizer by adding (adam=1) in the cfg file.

mAP = 41.09 %
loss = 1.5

The model has been training for 4 days, is there a way to speed it up by changing the Batch and Subdivisions?

**Comment by AlexeyAB** [comment]
@sharmarochan Set batch=64 and decrease subdivisions=.
Use better GPU, and compile Darknet with GPU=1 CUDNN=1

For increasing mAP - read: https://github.com/AlexeyAB/darknet#how-to-improve-object-detection

**Comment by sharmarochan** [comment]
Thank You for your response Alexey!!!
I have done certain changes will update you shortly as model is on training.
